htdocs/compta/facture/tpl/linkedobjectblock.tpl.php

```diff
@@ -85,7 +85,7 @@
     	}
 	}
     print '</td>';
-    print '<td class="linkedcol-statut right">'.$objectlink->getLibStatut(3).'</td>';
+    print '<td class="linkedcol-statut right">'.$objectlink->getLibStatut(3, $objectlink->getSommePaiement()).'</td>';
```

Comment:
I think $objectlink can be an object of any type and ->getSommePaiement does not exists for mots of them

Reply:
@eldy you're right, i added a method_exists() and did the same thing for supplier invoices